yarn.lock

```diff
@@ -4045,7 +4045,7 @@
     storybook-addon-outline "^1.4.1"
     ts-dedent "^2.0.0"
 
-"@storybook/addon-knobs@6.3.5":
+"@storybook/addon-knobs@6.3.0":
```

Comment:
was this intentional?

Reply:
not really, but it looks like its been resolved since 😌